Mathlib/Combinatorics/SimpleGraph/StronglyRegular.lean

```diff
@@ -170,8 +174,8 @@ theorem IsSRGWith.compl (h : G.IsSRGWith n k ℓ μ) :
     Gᶜ.IsSRGWith n (n - k - 1) (n - (2 * k - μ) - 2) (n - (2 * k - ℓ)) where
   card := h.card
   regular := h.compl_is_regular
-  of_adj := fun _v _w ha => h.card_commonNeighbors_eq_of_adj_compl ha
-  of_not_adj := fun _v _w hn hna => h.card_commonNeighbors_eq_of_not_adj_compl hn hna
+  of_adj := fun _ _ ha => h.card_commonNeighbors_eq_of_adj_compl ha
+  of_not_adj := fun _ _ hn hna => h.card_commonNeighbors_eq_of_not_adj_compl hn hna
```

Comment:
I think the `_v` spelling is preferred (and there's movement towards making `v` by itself legal)